spiffworkflow/variables.tf

```
@@ -0,0 +1,126 @@
+variable "cf_org_name" {
+  type        = string
+  description = "cloud.gov organization name"
+}
+
+variable "cf_space_name" {
+  type        = string
+  description = "cloud.gov space in which to deploy the apps"
+}
+
+variable "app_prefix" {
+  type        = string
+  description = "prefix to use for the three application names (<app_prefix>-[connector|backend|frontend])"
+  default     = "spiffworkflow"
+}
+
+variable "route_prefix" {
+  type        = string
+  description = "prefix to use for the application routes (<route_prefix>-connector.app.internal, <route_prefix>.apps.fr.cloud.gov[/api]); leave empty to generate"
+  default     = ""
+}
+
+variable "rds_plan_name" {
+  type        = string
+  description = "PSQL database service plan to use"
+  # See options at https://cloud.gov/docs/services/relational-database/#plans
+  default = "small-psql"
+}
+
+variable "rds_json_params" {
+  description = "A JSON string of arbitrary parameters"
+  type        = string
+  default     = null
+  # See options at https://cloud.gov/docs/services/relational-database/#setting-optional-parameters-1
+}
+
+variable "tags" {
+  description = "A list of tags to add to the module's resource"
+  type        = set(string)
+  default     = []
+}
+
+variable "process_models_repository" {
+  type        = string
+  description = "git repository with process models (for read-write, use SSH-style 'git@github.com:...' and supply your ssh_key)"
```

[suggestion] to avoid confusion about whether read-write is optional:

```suggestion
  description = "git repository with process models (use SSH-style 'git@github.com:...')"
```